lua/luasnip/nodes/snippet.lua

```diff
@@ -154,6 +154,8 @@ local function init_snippet_opts(opts)
 	-- return sn(t("")) for so-far-undefined keys.
 	in_node.stored = setmetatable(opts.stored or {}, stored_mt)
 
+	in_node.autotriggered = opts.autotriggered or false
```

Comment:
I think this option should be in the `context`-table (ie. the first one passed to `s`).
That is populated a few lines below, in `init_snippet_context` (again, docs missing :sweat_smile:)

Reply:
Ah already was at that point, that I didn't knew what to put in which options table. Yes we definitely can move this option to the `context`-table

Reply:
Solved in the most recent commit